core/modules/admin_bar/admin_bar.api.php

```diff
@@ -72,7 +72,7 @@ function hook_admin_bar_output_build(&$content) {
 
   // Add new top-level item to the menu.
   if (isset($content['menu'])) {
-    $content['menu']['myitem'] = array(
+    $content['menu']['menu']['myitem'] = array(
```

Comment:
As you rely on $content['menu']['menu'] here, you should also add it to the check above:

`isset($content['menu']['menu']`

Reply:
Thanks for the review, @indigoxela. Fixed.